paasta_tools/cli/cmds/remote_run.py

```diff
@@ -269,6 +285,13 @@ def add_subparser(subparsers: argparse._SubParsersAction) -> None:
         type=int,
         default=1800,
     )
+    start_parser.add_argument(
+        "-f",
+        "--follow",
+        help="Attach to and follow container output",
+        action="store_true",
+        default=False,
```

Comment:
i guess if folks don't use this the logs would be available through our  usual container logging pipeline?

that said, imo it's probably more likely that folks wanna see the output of commands :p

Reply:
Correct. I added this because as a replacement for local-run, it makes sense to have an option to see what stdout/stderr say, rather than having to rely on a separate tool tailing service logs.